src/components/MainContentCard.tsx

```diff
@@ -0,0 +1,17 @@
+import type { PropsWithChildren, ReactNode } from 'react';
+
+type Props = PropsWithChildren<{ header?: ReactNode; subheader?: ReactNode }>;
+
+const MainContentCard = ({ children, header, subheader }: Props) => {
+  return (
+    <div className="w-full bg-gradient-to-br from-[#fffcf2] to-[#ffffff] rounded-t-3xl rounded-b-xl shadow-[0px_34px_50px_0px_#ff7a1a]">
```

Comment:
I'm not familiar with this brackets in className syntax. Is some part of the build chain making classes on the fly?

Reply:
got it https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values